src/components/NavBottom.jsx

```diff
@@ -31,7 +31,7 @@ function NavBottom() {
           </Link>
         </div>
         <div className="CercleBtnNav">
-          <Link className="BtnLink" to="/login">
+          <Link className="BtnLink" to="/register">
```

Comment:
if it's the link to profile, it should point to /profile: it's up to the profile page to redirect to login if no user is connected (and it's up to login page to propose a link to register)